html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
```

Comment:
Nit: while you're dealing with my other complaints, none of the stuff in the head particularly needs to be indented.

Reply:
Header indentation removed.